explainers/subresource-loading.md

```diff
@@ -215,6 +215,17 @@ compressed](https://github.com/yoavweiss/url_compression_experiments).
 
 ## Alternate designs
 
+### Resource Bundles
```

Comment:
Remember to update the table of contents. I use https://marketplace.visualstudio.com/items?itemName=AlanWalk.markdown-toc to do that automatically.

Reply:
A good point. I forgot it. Let me update the TOC, then merge this PR. Thanks!